Makefile

```diff
@@ -19,29 +23,38 @@ KUTTL_REPO = https://github.com/kudobuilder/kuttl
 PROJECT = github.com/shipwright-io/image
 GEN_OUTPUT = /tmp/$(PROJECT)/infra/images
 
+# destination namespace to install target
+NAMESPACE ?= shipwright-build
+
+# the container image produced by ko will use this repostory, and combine with the application name
+# being compiled
+KO_DOCKER_REPO ?= $(REGISTRY_HOSTNAME)/$(REGISTRY_USERNAME)
+
+# golang flags are exported through the enviroment variables, reaching all targets
+GOFLAGS ?= -v -mod=vendor -ldflags='-Xmain.Version=$(VERSION)'
+
+.EXPORT_ALL_VARIABLES:
```

Comment:
@ricardomaraschini it's importante to highlight this bit, so it makes all Makefile variable automatically environment variables.